commands/core/config.drush.inc

```diff
@@ -444,10 +444,10 @@ function _drush_config_export($destination, $destination_dir, $branch) {
 
     $config_comparer = new StorageComparer($comparison_source, $target_storage, Drupal::service('config.manager'));
     if (!$config_comparer->createChangelist()->hasChanges()) {
-      return drush_log(dt('There are no changes to export.'), LogLevel::OK);
+      return drush_log(dt('The active config does not differ to the export directory (!target).', array('!target' => $destination_dir)), LogLevel::OK);
     }
 
-    drush_print("The following configuration changes have been made since the last export:\n");
+    drush_print("Differences of the active config to the export directory:\n");
```

Comment:
I'm not sure that this is an improvement. Anyone else have an opinion or suugestion?